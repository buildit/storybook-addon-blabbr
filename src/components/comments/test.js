import React from 'react';
import renderer from 'react-test-renderer'; // eslint-disable-line
import Comments from './';

test('Comments render correctly', () => {
  const tree = renderer.create(
    <Comments
      comments={[]}
      commentIdBeingEdited="commentId123"
      userCommentBeingUpdated="updating comment"
      currentUser="currentUser"
      handleEditUserComment={() => {}}
      handleEditUserCommentChange={() => {}}
      handleEditUserCommentSubmit={() => {}}
      handleEditUserCommentCancel={() => {}}
      handleDeleteUserComment={() => {}}
      handleShowAllComments={() => {}}
      isShowingAllComments
    />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
