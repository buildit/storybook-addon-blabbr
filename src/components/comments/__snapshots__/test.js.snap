// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments render correctly 1`] = `
<div>
  <div>
    <p>
      No comments to show for this story
    </p>
  </div>
</div>
`;
